packages/salesforcedx-vscode-core/src/conflict/conflictView.ts

```diff
@@ -48,12 +49,17 @@ export class ConflictView {
     title: string,
     remoteLabel: string,
     reveal: boolean,
-    diffResults?: DirectoryDiffResults
+    diffResults?: DirectoryDiffResults,
+    differences?: boolean
   ) {
+    this.diffsOnly = differences || false;
```

Comment:
```suggestion
    diffsOnly = false
  ) {
    this.diffsOnly = diffsOnly;
```

nit - the name `differences` feels like it actually is the diffs